sklearn/dummy.py

```diff
@@ -77,7 +77,7 @@ class DummyClassifier(BaseEstimator, ClassifierMixin, MultiOutputMixin):
         Is automatically set to True if the input y is passed in sparse format.
     """
 
-    def __init__(self, strategy="stratified", random_state=None,
+    def __init__(self, strategy="prior", random_state=None,
```

Comment:
The change is not that easy. We cannot change the default now. We have to warn user that the default will change in 2 versions.

The pattern to follow is documented there: https://scikit-learn.org/stable/developers/contributing.html#change-the-default-value-of-a-parameter